src/test/java/com/redhat/devtools/lsp4ij/fixtures/LSPFoldingRangeFixtureTestCase.java

```diff
@@ -167,6 +200,9 @@ private static List<TextRange> getExpectedTextRanges(@NotNull String fileBody) {
             int endOffset = endOffsets.get(i);
             expectedTextRanges.add(TextRange.create(startOffset, endOffset));
         }
+
+        // Sort for stability
+        expectedTextRanges.sort(TextRangeUtil.RANGE_COMPARATOR);
```

Comment:
As mentioned above, sorting for consistency and stability so that we can have parallel lists of mock folding ranges and fold region text ranges.